torch/_inductor/constant_folding.py

```diff
@@ -163,11 +167,13 @@ def run(self):
 
 
 @torch.utils._python_dispatch._disable_current_modes()
-def constant_fold(gm):
+def constant_fold(gm, filter_fn: Optional[Callable[[torch.fx.Node], bool]] = None):
```

Comment:
Please have someone from inductor, who touched this file before, also review this part of the code.